code/mathematical-algorithms/gcd_and_lcm/gcd_and_lcm.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace gcd_and_lcm
+{
+    class gcd_lcm
```

Comment:
Instead of having to create an object every time you want the gcd/lcm of two numbers, make the methods in this class static, let them accept two parameters a and b, and remove the private variables and the constructor.

Reply:
I didn't see this. 2nd time collision 😅